### ISSUE TITLE
allow to specify format base in workspace.json now that github default branch is `main`

### ISSUE THREAD

**Issue by cafesanu** [nrwl/nx#3732]
## Description

Github recently changed the default name of the base branch from `master` to `main`. https://github.com/github/renaming#on-october-1-2020-newly-created-repositories-will-default-to-main

When I run `nx format:check` it does fail since the base `master` does not exist. I know that I can pass it via `--base main`, but it would be super nice if we can specify the base branch in our `workspace.json` file.

Thanks!
## Motivation

Basically any new project in github will fail and this problem will become more obvious with the pass of time.

## Suggested Implementation

Add an option in `workspace.json` to specify base branch.

**Comment by vsavkin** [comment]
Thank you for submitting the issue!

We added an option to configure default base a couple of months ago: https://nx.dev/react/workspace/configuration#nx-json

Is it what you are asking for?

**Comment by cafesanu** [comment]
wow I totally missed that. That is exactly what I am looking for! Thanks and closing issue :)
